src/common/__tests__/Avatar-test.js

```diff
@@ -0,0 +1,11 @@
+import 'react-native';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Avatar from '../Avatar';
+
+it('renders correctly', () => {
+  const tree = renderer.create(
+    <Avatar />
```

Comment:
Thanks for the feedback, Neeraj. I will start working on the MessageView testing today.